filters-share/extended_css_ELEMHIDE.txt

```diff
@@ -1,3 +1,9 @@
+ppss.kr###custom_html-3
+ppss.kr###genesis-content > div,span:not(:has(header)):not([itemprop])
+ppss.kr###ppss_wing_banner_left
+ppss.kr##.ad-video
+ppss.kr##.brand-margin
```

Comment:
The class name doesn't look like ad but without this, we'll face huge external link.